package.json

```diff
@@ -144,7 +144,7 @@
     "@angular/platform-browser": "^7.2.2",
     "@angular/platform-browser-dynamic": "^7.2.2",
     "@angular/router": "^7.2.2",
-    "@terminus/ngx-tools": "latest",
+    "@terminus/ngx-tools": "^5.2.0",
```

Comment:
This should be "latest"
I understand their were some issues last week, but we now have a work-around.

Reply:
Once it's changed back to `latest` we should be able to run `yarn upgrade @terminus/ngx-tools` to update the lock file so that `5.2.0` is installed